mods/voidmulti.sc2mod/base.sc2data/gamedata/modeldata.xml

```diff
@@ -852,6 +880,10 @@
     <CModel id="TemporalFieldImpactModel">
         <Model value="Assets\Effects\Protoss\TimeWarpImpact_Red\TimeWarpImpact_Red.m3"/>
     </CModel>
+    <CModel id="Ultralisk">
+        <ScaleMax value="0.750000,0.750000,0.750000"/>
+        <ScaleMin value="0.750000,0.750000,0.750000"/>
```

Comment:
**bug**: Ultralisk is the wrong model entry to edit.
=> FIX: UltraliskBase should be edited, so all skins are edited as well! Then check if all inherit it or if any overrides anything.

Ultralisk model's scale was reduced from 0.85 to 0.75 (-11.76%).
Patch notes state: reduced size by 12.5%.

It looks fine in the game. Naturally you want to make the model as big as possible, so the attack animations still cover the area it damages.
The size looked fine (but I did not test it vs a big pile of small units, yet)